src/command_helpers.rs

```diff
@@ -40,6 +42,13 @@ impl CargoOutput {
         }
     }
 
+    pub(crate) fn print_debug(&self, arg: &dyn Display) {
+        if self.metadata {
+            println!("cargo:rerun-if-env-changed=CC_ENABLE_DEBUG_OUTPUT");
```

Comment:
Nit:

I recommend to add a `AtomicBool` to `CargoOutput::new` to prevent this from being printed multiple times.

Printing isn't exactly cheap, though it's also **not mandatory** for this PR to be merged.

Reply:
IMO the most important thing here is the documentation, feel free to ignore this if you don't have time/effort to do this.

Reply:
I added a `checked_dbg_var` field that's an `Arc<AtomicBool>`, does that work? It would only check if `self.metadata` is true and this field hasn't been set yet, print out the `rerun-if-env-changed` bit, and then not print again